.eslintrc

```diff
@@ -4,7 +4,7 @@
     "ecmaVersion": 2018,
     "sourceType": "module",
     "project": "./.eslint.tsconfig.json",
-    "allowAutomaticSingleRunInference": true
+    "allowAutomaticSingleRunInference": false,
```

Comment:
Have seen an issue where typescript eslint rules report incorrectly when this is turned on, it does not happen on manual `yarn run eslint` runs but it does happen in `lint-staged`.